format_and_overlap_determination.md

```diff
@@ -0,0 +1,118 @@
+
+## Definitions ##
+**Format**
+: mono (spm), multi (mpm), serial (ser)
+
+**Cluster**
+: Collection of HT Items, Holdings, and Concordance pairs united by a list of OCNs determined to be equivalent.
+
+**Bib Record**
+: A preferred record from the Zephir clustering system containing one or more HT Items. Has a single Hathitrust record ID.
+
+**HT Item**
+: Volume ID and associated metadata taken from the Hathifiles. 
+
+**Print Serials List**
+: List of Record Ids determined to be serials by some unknown UMich process. List is limited to those held by UMich.
+
+**Large Clusters**
+: Clusters large enough that they create implementation problems. Identified by an OCN, there are currently 11 on the list.
+
+**Billing Entity**
+: The `billing_entity` field taken from the `ht_collections` table associated with a particular HT Item through the collection code. Used for cost allocation when holdings are incomplete or no clustering can be performed. Also has subtle and likely inconsequential implications for items contributed by UCM and KEIO.
+
+**n_enum**
+: A normalized enumeration. Description fields are processed for enumeration and chronology. Only enumeration ends up being used. 
+
+
+## Item Format ##
+
+**Serial**
+: Items with a **Bib Record ID** found on the **Print Serials List**, or in a **Large Cluster** 
+
+**Multi-part Monograph**
+: Is not a Serial. Shares a **Bib Record** with an item that has a non-empty `n_enum`. [^1]
+
+**Single-part Monograph**
+: Is not a Serial or Multi-part
+
+**NB**: A cluster may contain items of multiple formats.  
+**NB2**: Format as reported by members is irrelevant.
+
+## Cluster Format ##
+
+**Multi-part Monograph**
+: Any of the items in the cluster are MPMs.
+
+**SER / SPM**
+: Is not MPM. One or more of the items in the cluster are SPM _and_ one or more of the items in the cluster are SER.
+
+**SER**
+: Is not MPM or SER/SPM. Any of the items in the cluster are SER (and by implication ALL are SER).
+
+**SPM**
+: Is not MPM, SER/SPM, or SER.
+
+## Cost Allocation ##
+
+**Target Cost**
+: Set by the Board and Executive Director. Portion of the budget to be paid for by item allocations.
+
+**Cost Per Volume**
+: Target Cost / Total Number of HT Items
+
+**Member Weight**
+: A number assigned to a member by the ED that determines their share of Public Domain costs. 
+
+**Total Weight**
+: The sum of member weights found in the `ht_billing_members` table of the holdings database. **Which process/system has ownership of this table?**
+
+**Public Domain Cost**
+: Cost Per Volume * Number of Public Domain Volumes (see Public Domain vs In-Copyright)
+
+**Public Domain Cost for Member**
+: Total Public Domain Cost / Total Weight * Member Weight
+
+**Extra Per Member**
+: The IC Costs assigned to Hathitrust / Number of members found in the `ht_billing_members` table that aren't Hathitrust. Hathitrust has IC costs due to being the billing entity for some items.
+
+**HScore**
```

Comment:
I might expand on this - the share is the inverse of the number of members who report holding the item

Reply:
:+1: